docs/concepts/macros/sqlmesh_macros.md

```diff
@@ -1210,6 +1210,27 @@ SELECT
 FROM table
 ```
 
+#### Vararg and kwarg arguments
```

Comment:
I'd go for `#### Variable-length and named arguments` or something, and then establish the short-hand terminology using parentheses.